lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -258,10 +264,37 @@ def get_diff_image(self):
                 return png.getvalue()
 
     def handle_event(self, event):
+        # Check if all pending images have been processed.
+        if self._num_received_images is not None:
+            pending_images = (
+                self.manager._num_sent_images > self._num_received_images
+                )
```

Comment:
This fits in one line
```suggestion
            pending_images = self.manager._num_sent_images > self._num_received_images
```